guides/technical-guides/git.md

```diff
@@ -4,6 +4,14 @@ This guide is meant to set up some handy git configuration options and/or tools
 that should help you get set up quickly. This section will need to be filled in
 more for other operating systems as and when available.
 
+## Flight Rules
+
+Flight Rules fantastic GitHub repository which has a plethora of information
```

Comment:
Is there a verb/article missing here? _"Flight Rules **is a** fantastic…”_…?

Reply:
![image](https://user-images.githubusercontent.com/106537794/215426450-701163da-877c-4481-be92-b2ffcad9b2ce.png)

Good spot! Fixed :)